proto/vtctldata.proto

```diff
@@ -86,14 +86,20 @@ message Workflow {
   ReplicationLocation source = 2;
   ReplicationLocation target = 3;
   int64 max_v_replication_lag = 4;
-  map<string, ShardReplicationStatus> shard_statuses = 5;
+  map<string, ShardStream> shard_streams = 5;
 
   message ReplicationLocation {
     string keyspace = 1;
     repeated string shards = 2;
   }
 
-  message ReplicationStatus {
+  message ShardStream {
+    repeated Stream streams = 1;
+    repeated topodata.Shard.TabletControl tablet_controls = 2;
+    bool is_primary_serving = 3;
+  }
+
+  message Stream {
```

Comment:
@rohit-nayak-ps let me know what you think of the new naming

Reply:
This is good!